README.md

```diff
@@ -0,0 +1,87 @@
+# Hopsan
```

Comment:
A hint to the compiled releases may be helpful. But that would be only temporary.